### PR TITLE
refactor(css design tokens): typography line heights + font size from px to rem unit

### LINKED ISSUE #30: design: forbid negative line-height
**Subject**

Negative line-height should be forbidden.
(WCAG recommends at least 1.5)

Right now we have this for large titles :

image

This will break readability on multilines titles 
See: https://codepen.io/__sch__/pen/QWKaMmX

### PR DESCRIPTION
**Pull request by lauthieb**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Make sure you are making a pull request against the **main branch** (left side).
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Description

This Pull Request fixes #30 & #29. I have also replaced `px` units on typography's design tokens with a relative unit: `rem` (see #19).

I've used official values now validated by designers available here: https://www.decathlon.design/726f8c765/p/860e14-typography

Thanks in advance for your review! :)

## Does this introduce a breaking change?

- No

### CODE REVIEW

File: packages/sources/css/src/design-tokens/src/typography.css [hunk 1]
```diff
@@ -21,119 +21,119 @@
...
-  --vtmn-typo_title-1-font-size: 40px;
-  --vtmn-typo_title-2-font-size: 36px;
-  --vtmn-typo_title-3-font-size: 28px;
-  --vtmn-typo_title-4-font-size: 24px;
-  --vtmn-typo_title-5-font-size: 20px;
-  --vtmn-typo_text-1-font-size: 18px;
-  --vtmn-typo_text-2-font-size: 16px;
-  --vtmn-typo_text-3-font-size: 14px;
+  --vtmn-typo_display-1-font-size: 3.75rem;
+  --vtmn-typo_display-2-font-size: 2.625rem;
```
Comment (NicolasBreunin) [inline packages/sources/css/src/design-tokens/src/typography.css:25]: is'nt it '2rem' ? 
Capture d’écran 2021-01-15 à 14 42 20

File: packages/sources/css/src/design-tokens/src/typography.css [hunk 2]
```diff
@@ -21,119 +21,119 @@
...
-  --vtmn-typo_display-2-line-height: 30px;
-  --vtmn-typo_title-1-line-height: 42px;
-  --vtmn-typo_title-2-line-height: 40px;
-  --vtmn-typo_title-3-line-height: 27px;
-  --vtmn-typo_title-4-line-height: 28px;
-  --vtmn-typo_title-5-line-height: 24px;
-  --vtmn-typo_text-1-line-height: 26px;
-  --vtmn-typo_text-2-line-height: 24px;
-  --vtmn-typo_text-3-line-height: 22px;
+  --vtmn-typo_display-1-line-height: 1;
```
Comment (NicolasBreunin) [inline packages/sources/css/src/design-tokens/src/typography.css:41]: 1 what ? 1 carott ? 1 potatoe ? 😀

File: packages/sources/css/src/design-tokens/src/typography.css [hunk 3]
```diff
@@ -21,119 +21,119 @@
...
 
   :root {
-    --vtmn-typo_display-1-line-height: 60px;
-    --vtmn-typo_display-2-line-height: 44px;
-    --vtmn-typo_title-1-line-height: 48px;
-    --vtmn-typo_title-2-line-height: 42px;
-    --vtmn-typo_title-3-line-height: 34px;
-    --vtmn-typo_title-4-line-height: 28px;
-    --vtmn-typo_title-5-line-height: 24px;
+    --vtmn-typo_display-1-line-height: 1;
```
Comment (NicolasBreunin) [inline packages/sources/css/src/design-tokens/src/typography.css:87]: same 😀

File: packages/sources/css/src/design-tokens/src/typography.css [hunk 4]
```diff
@@ -21,119 +21,119 @@
...
 
   :root {
-    --vtmn-typo_display-1-line-height: 76px;
-    --vtmn-typo_display-2-line-height: 56px;
-    --vtmn-typo_title-1-line-height: 50px;
-    --vtmn-typo_title-2-line-height: 42px;
-    --vtmn-typo_title-3-line-height: 42px;
-    --vtmn-typo_title-4-line-height: 32px;
-    --vtmn-typo_title-5-line-height: 24px;
+    --vtmn-typo_display-1-line-height: 1;
```
Comment (NicolasBreunin) [inline packages/sources/css/src/design-tokens/src/typography.css:131]: same 😀

File: packages/sources/css/src/design-tokens/src/typography.css [hunk 5]
```diff
@@ -21,119 +21,119 @@
...
```
Comment (NicolasBreunin) [inline packages/sources/css/src/design-tokens/src/typography.css:109]: same 😀

File: packages/sources/css/src/design-tokens/src/typography.css [hunk 6]
```diff
@@ -21,119 +21,119 @@
...
 
   :root {
-    --vtmn-typo_display-1-line-height: 60px;
-    --vtmn-typo_display-2-line-height: 44px;
-    --vtmn-typo_title-1-line-height: 48px;
-    --vtmn-typo_title-2-line-height: 42px;
-    --vtmn-typo_title-3-line-height: 34px;
-    --vtmn-typo_title-4-line-height: 28px;
-    --vtmn-typo_title-5-line-height: 24px;
+    --vtmn-typo_display-1-line-height: 1;
```
Comment (NicolasBreunin) [inline packages/sources/css/src/design-tokens/src/typography.css:65]: same 😀

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 2 above)
Comment (lauthieb) [reply packages/sources/css/src/design-tokens/src/typography.css:41]: Please see #29

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 3 above)
Comment (lauthieb) [reply packages/sources/css/src/design-tokens/src/typography.css:87]: Please see #29

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 4 above)
Comment (lauthieb) [reply packages/sources/css/src/design-tokens/src/typography.css:131]: Please see #29

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 5 above)
Comment (lauthieb) [reply packages/sources/css/src/design-tokens/src/typography.css:109]: Please see #29

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 6 above)
Comment (lauthieb) [reply packages/sources/css/src/design-tokens/src/typography.css:65]: Please see #29

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 1 above)
Comment (lauthieb) [reply packages/sources/css/src/design-tokens/src/typography.css:25]: Font sizes has changed too. Please see https://www.decathlon.design/726f8c765/p/860e14-typography

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 2 above)
Comment (NicolasBreunin) [reply packages/sources/css/src/design-tokens/src/typography.css:41]: yes sorry Flavian hacked my computer

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 3 above)
Comment (NicolasBreunin) [reply packages/sources/css/src/design-tokens/src/typography.css:87]: i saw :)

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 1 above)
Comment (NicolasBreunin) [reply packages/sources/css/src/design-tokens/src/typography.css:25]: OK 👍

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 2 above)
Comment (lauthieb) [reply packages/sources/css/src/design-tokens/src/typography.css:41]: Ahah!

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 3 above)
Comment (lauthieb) [reply packages/sources/css/src/design-tokens/src/typography.css:87]: So is it ok for you? :)

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 5 above)
Comment (NicolasBreunin) [reply packages/sources/css/src/design-tokens/src/typography.css:109]: same 😀

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 4 above)
Comment (NicolasBreunin) [reply packages/sources/css/src/design-tokens/src/typography.css:131]: don't blame me 🥰

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 6 above)
Comment (NicolasBreunin) [reply packages/sources/css/src/design-tokens/src/typography.css:65]: or not again 😀

**Review by NicolasBreunin — APPROVED** [review]

File: packages/sources/css/src/design-tokens/src/typography.css (hunk 1 above)
Comment (lauthieb) [reply packages/sources/css/src/design-tokens/src/typography.css:25]: Thanks @NicolasBreunin !

**Review by AgatheGossey — APPROVED** [review]

**Review by RomainJoly — APPROVED** [review]

**Review by MatthiasCartelDkt — COMMENTED** [review]
Good for me :)

**Pull request merged**